yarn.lock

```diff
@@ -173,6 +173,13 @@
   dependencies:
     "@babel/types" "^7.10.4"
 
+"@babel/helper-module-imports@^7.12.13":
```

Comment:
If possible, dedupe with the above `@babel/helper-module-imports@^7.10.4`.  Same goes for most of the added dependencies in this PR.

Reply:
https://github.com/Shopify/web-configs/pull/207